src/turbo_response/template.py

```diff
@@ -27,15 +28,17 @@ def render_turbo_stream_template(
     return render_turbo_stream(
         action,
         target,
-        render_to_string(
-            template,
-            {
-                **(context or {}),
-                "turbo_stream_target": target,
-                "turbo_stream_action": action.value,
-                "is_turbo_stream": True,
-            },
-            **template_kwargs,
+        mark_safe(
```

Comment:
I would push this `mark_safe` into the `render_turbo_stream` function rather than require the caller to do it.

Reply:
Do you mean with an optional argument ("is_safe")?